packages/manager/apps/pci-instances/src/data/hooks/instance/selectors/instances.selector.ts

```diff
@@ -120,21 +120,27 @@ const mapInstanceAddresses = (instance: TAggregatedInstanceDto) =>
     return acc.set(type, [rest]);
   }, new Map<TInstanceAddressType, TAggregatedInstanceAddress[]>());
 
+const isActionToDisplay = (name: TActionName) => name !== 'vnc';
```

Comment:
We must find later another way to achieve this and thus to prevent from having this business rule into the client logic ;)